### DISCUSSION TITLE
Lingmo OS 2.0 Hydrogen Released!

### DISCUSSION THREAD

**Discussion by elysia-best** [LingmoOS/Community#19] [Announcements]
We are happy to announce that Lingmo OS 2.0 is finally released!

You can download the latest release right on the [Sourceforge](https://sourceforge.net/projects/lingmo-os/files/latest/download)

Below is the release note for this version.
 
</html><h1 id="lingmo-os-2-0-hydrogen-release-notes">Lingmo OS 2.0 Hydrogen Release Notes</h1>
<p><strong>Note</strong>: <em>2.0 (Hydrogen) is a transitional version, so it does not add too many features, please wait for 3.0 (Helium) which will be released in a month.</em></p>
<p><strong>life cycle:</strong></p>
<ul>
<li>2.0 (Hydrogen) is a transitional version, so its support period is only one month, but we recommend that users install it because it can accept all updates after this version.</li>
</ul>
<p><strong>What&#39;s new?</strong></p>
<ul>
<li>Kernel version upgraded to <strong><em>6.10.9</em></strong></li>
<li>Merge security updates and latest packages from upstream <strong>Debian GNU/Linux</strong></li>
<li>Better visual effects and experience</li>
<li>Supports the latest firmware versions</li>
<li>System Version upgraded to <strong><em>2.0</em></strong></li>
<li>Codename upgraded to <strong><em>Hydrogen</em></strong> </li>
</ul>
<p><strong>Fixed bugs:</strong></p>
<ul>
<li>Fixed an issue where the package could not be updated</li>
<li>Fixed the issue where the software store could not be used</li>
<li>Fixed the crash issue on some pages</li>
</ul>
<p><strong>What&#39;s next?</strong></p>
<ul>
<li>Complete File Manager Functionality</li>
<li>System Explorer</li>
<li>System picture browser</li>
<li>System Package Installer</li>
<li>Complete system update function</li>
<li>Complete desktop functions</li>
<li>Desktop Widget</li>
<li>Working area</li>
<li>Complete starter function and full/half screen switching</li>
<li>Drag the USB flash drive icon to the recycle bin icon to pop up the device (?</li>
</ul>
<p><strong>Lingmo OS Development Team</strong>
<a href="[lingmo.org](https://packages.lingmo.org/lingmo/release/stable/hydrogen/release-note/en/lingmo.org)">lingmo.org</a></p>

**Comment by celiopy** [comment]
Hello, I just done a fresh install, when I tried to update, I got: "Not found". When I try to install anything with apt, also not found, is that intentional?

```
Err:1 https://packages.lingmo.org/lingmo hydrogen/main amd64 cups-pk-helper amd64 0.2.6-2
  404  Not Found [IP:  443]
Err:2 https://packages.lingmo.org/lingmo hydrogen/main amd64 hyperv-daemons amd64 6.11-1
  404  Not Found [IP: 443]
Err:3 https://packages.lingmo.org/lingmo hydrogen/main amd64 linux-libc-dev all 6.11-1
  404  Not Found [IP: 443]
```

**Reply by elysia-best** [reply]
Hi caliopy. We are now changing our mirror to debian's official mirror. Thus you can change to Debian 13 Trixie's mirrors to download packages.

**Comment by angrycsm** [comment]
Sorry, I don't know how to speak English, I'm using a translator, I recently installed the version and my SUPER or WINDOWS key to open the applications screen doesn't work, how can I solve this? It's some keybind error, I tested other Linux versions, and they work with the SUPER key to show the applications tab, I didn't want to have to keep clicking with the mouse to open the applications.

**Comment by luciano1302censuratus** [comment]
Hello everyone, I'm trying to install lingmo-os-2.0-hydrogen-release-desktop-amd64.iso on my old LG A410 4GB RAM notebook but I can't. I wanted to know if it runs on MBR system, that is, LEGACY mode? I don't have a PC (it burned after a lightning storm!) and at the moment I have no way to fix my PC. This Notebook is the only computer I have. I would like to have a very light system!

**Reply by elysia-best** [reply]
You can try to use the nightly build here https://github.com/LingmoOS/live-build-config/actions/runs/12126291625/artifacts/2263514142 

However, please keep mind that this iso was based on Debian testing. Be careful not to uninstall system package accidentally, or you'll get a broken desktop :/

**Reply by samwhelp** [reply]
> English is not my mother tongue, i just try to explain as i can.


## Reply

@luciano1302censuratus

> I wanted to know if it runs on MBR system, that is, LEGACY mode?

Yes, it can run on MBR system, LEGACY mode.

It worked on my notebook (MBR system, LEGACY mode).


## Notice

* You should `disable uefi support` on bios before booting
* Using Calamares installer, Partition must leave 1MB size, and set flag to `bios_grub` when grub install on (MBR system, LEGACY mode).




## Sample Command

> This is sample command for creating partition installing MBR system.

``` sh
sudo parted --script -- "/dev/sdc" \
	mktable gpt \
	mkpart primary '0%' '2M' \
	mkpart primary '2M' '100%' \
	set 1 bios_grub on \
	print
```

> [My article](https://samwhelp.github.io/note-about-grub/read/howto/boot-iso/create-live-usb-disk-for-bios.html), but it's not English.




## Another Way for test

May be you can try to install Debian Bookworm,

to check it worked on (MBR system, LEGACY mode).

Debian Bookworm worked on my notebook (MBR system, LEGACY mode).
